src/Autofac/Util/ReflectionExtensions.cs

```diff
@@ -121,4 +121,56 @@ public static bool TryGetDeclaringProperty(this ParameterInfo pi, [NotNullWhen(r
 
         return callExpression.Constructor!;
     }
+
+    /// <summary>
+    /// Checks if <paramref name="memberInfo"/> has a <c>RequiredMemberAttribute</c>.
```

Comment:
```suggestion
    /// Checks if a provided member has a <c>RequiredMemberAttribute</c>.
```

Trying to avoid referencing parameters before they're defined in the doc.

Reply:
While I dont think xmldocs have a concept of ordering, sure ;)